example/example/autoextract.py

```diff
@@ -30,6 +30,7 @@ def __call__(self):
 
 @inlineCallbacks
 def get_autoextract_product(url):
+    raise returnValue({'product': {'url': url, 'name': url}})
```

Comment:
Could you please remove this change? I see how it was useful for testing, but let me send you an API key instead.

Reply:
Sure, was using it just for tests. Was expecting an initial review before removing it.